### ISSUE TITLE
Screen Reader: Global: Button 'Scan receipt' is announced with a redundant 'floating action'

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77363]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack Android, open the app
2. On the Inbox screen, swipe right to focus the Scan receipt (camera) button and note how it is announced

This is a global issue occurring in all bottom navigation bar tabs

## Expected Result:
Screen readers should announce the with am meaningful and concise label, e.g.:
"Scan receipt, button".

## Actual Result:
The button is announced redundantly as "Scan receipt, floating action, button" while "Floating action" adds no meaningful information.

## Area issue was found in:
Inbox

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Remove the redundant text from the element's accessibilityLabel that causes the "floating action" announcement.
Ensure the element is announced with a concise and meaningful label: "Scan receipt, button".

Refer to:
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment(s)
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/d8e876e2-b183-46ca-b6ca-219b085ffd49)

https://github.com/user-attachments/assets/6a3f62aa-56d9-4e55-99dc-9c39f743fd80

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005833
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
fixed https://github.com/Expensify/App/issues/79233
